src/bitcoin-cli.cpp

```diff
@@ -294,6 +294,8 @@ int CommandLineRPC(int argc, char *argv[])
     catch (const boost::thread_interrupted&) {
```

Comment:
Sorry, yes you mention it there. It just looked strange to me to add something instead of replace.